Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM arxiv/base:latest
 
 WORKDIR /opt/arxiv/
 
-RUN yum install -y which mariadb-devel
+RUN yum install -y which mariadb-devel sqlite
```

Comment:
Is the dual-compatability of the container with mariadb and sqlite a desired feature or an artifact of my backend switch?

Reply:
Looks like an artefact. But will leave it in case you want to re-enable your local workflow.